.gitignore

```diff
@@ -1,2 +1,6 @@
 /vendor/
-/composer.phar
\ No newline at end of file
+composer.*
+.vagrant/*
+/nbproject/*
+output/*
+Cached_Cdm_files/*
```

Comment:
These changes to .gitignore are pretty machine-specific. Were they included in the PR intentionally?